markdown_it/common/normalize_url.py

```diff
@@ -54,7 +54,7 @@ def unescape_string(s):
 
 
 def normalize_uri(uri):
-    return quote(uri.encode("utf-8"), safe=str("/@:+?=&()%#*,"))
+    return quote(uri, safe="/@:+?=&()%#*,")
```

Comment:
so why exactly does this work? does quote accept bytes or strings, and why wrap the safe text in `str`?

Reply:
> so why exactly does this work? does quote accept bytes or strings

Yeah `quote()` can take a string and do `utf-8` encoding internally. If we for some reason wanted to pass in `bytes`, then there is `quote_from_bytes` for that.

> and why wrap the safe text in str

No reason really, this PR removes the wrapping.